index.html

```diff
@@ -643,12 +643,15 @@ <h2>Security</h2>
           </ul>
         </div>
         <p><span class="rfc2119-assertion" id="common-constraints-security-2">
-            Conformant Consumers MUST support all of these security schemes.</span>
+            Conformant Consumers MUST support at least all of these security schemes.</span>
```

Comment:
Do you mean...

Consumers MUST support at least **one** of these security schemes?

IF not.. "at least all" sounds strange, then I would rather stick to "all"

Reply:
Consumers must support all of the security schemes in order to guarantee interoperability with all Things using the profile.

I agree the existing wording made more sense.

Reply:
@danielpeintner 
We discussed it in today's call and agreed to merge it. If there's a need to improve the text, please create a new issue.